include/ofi_iov.h

```diff
@@ -171,6 +171,10 @@ ofi_iov_within(const struct iovec *iov1, const struct iovec *iov2)
 
 void ofi_consume_iov(struct iovec *iovec, size_t *iovec_count, size_t offset);
 
+void ofi_consume_iov_desc(struct iovec *iovec, void **desc, size_t *iovec_count, size_t offset);
```

Comment:
Please check line lengths here and in .c file and limit to around 80 characters.

Reply:
Fixed in new revision.